docs/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenAPI 3 support
+Currently no automatic OpenAPI 3 spec generation is supported. We are waiting for updates in
+[swaggo](https://github.com/swaggo/swag/issues/548).
+
+## Temporary manual solution
+1. Generate Swagger 2 spec using `swag init`.
+2. Convert Swagger 2 API json string ([docs.go](docs.go)) to OpenAPI 3 using: <https://mermade.org.uk/openapi-converter>.
+3. Convert output yaml to json, e.g. using: <https://onlineyamltools.com/convert-yaml-to-json>.
+4. Put result into the source ([docs.go](docs.go)) and update `"servers"` field with: `"servers": [\{"url": "http://localhost:8080"}]`.
```

Comment:
Please create a script to automate these tasks

Reply:
It would be much more difficult. These links are not APIs but web GUIs. I think it's fair enough as a temporary solution for this time. We can automate if there will be no other way but I hope that some support for OpenAPI 3 will be added to swaggo and we will use it.

Reply:
I've checked the tool and it seems like there is an API available - http://petstore.swagger.io/?url=https://mermade.org.uk/openapi-converter/contract/swagger.json#/. I believe we can automate this task by converting the apec using this API?

Reply:
I've added test for OpenAPI3 docstring validation.